frontend/dockerfile/dockerfile2llb/convert.go

```diff
@@ -2278,8 +2229,7 @@ func reportUnmatchedVariables(cmd instructions.Command, buildArgs []instructions
 	if len(unmatched) == 0 {
 		return
 	}
-	options := metaArgsKeys(opt.metaArgs)
```

Comment:
@daghack I didn't replace this with `globalArgs`. Afaics the valid options need to be in the `env` . If they are defined in the global section but not available in the `env` then they can't be really used yet.